tests/test_load.py

```diff
@@ -1648,6 +1651,39 @@ def test_load_from_disk_with_default_in_memory(
         _ = load_from_disk(dataset_path)
 
 
+@contextmanager
+def moto_server():
+    with patch.dict(os.environ, {"AWS_ENDPOINT_URL": "http://localhost:5000"}):
```

Comment:
You might have to add some dummy creds here to get past the NoCredentialsError in the test http://docs.getmoto.org/en/latest/docs/getting_started.html#recommended-usage

Reply:
thanks, yes this is correct. I setup a docker container to reproduce and fix. I also decided to minimize the change to hopefully avoid unintended consequences. 🤞 

Pushed, please have a look when you can.